### ISSUE TITLE
Location cone is on STL

### ISSUE THREAD

**Issue by coinnodenl** [skarkkai/touch-mapper#27]
The location you enter will have a cone, which is on the STL.

**Comment by mgrundkoetter** [comment]
Would be nice to have this removable (maybe as advanced option).

**Comment by Dickmaulruessler** [comment]
> Would be nice to have this removable (maybe as advanced option).

Yes, that would be great!

**Comment by sirnightowl** [comment]
Agreed, this would be absolutely perfect without the cone. I think it saids "Show advanced options (also hides location marker)" so maybe it's a bug that it doesn't?

**Comment by dancingbear1965** [comment]
Is there any movement of this issue? Whilst the rendered image does not show the cone, the STL file does. So it seems the 'hide location marker' aspect is not working.

**Comment by daviddoria** [comment]
I also ran into this problem. It would be great to be able to remove the cone from the STL.

**Comment by HJADR** [comment]
+1 for this issue.

**Comment by Plaethe** [comment]
+1 :-)

**Comment by Antwelm** [comment]
The cone is also much higher than any other feature on the map. Definition of sore thumb. Nice for those that need it, but pretty annoying for everyone else.

**Comment by skarkkai** [comment]
I hear you people, I will remove this at some point. Thanks for letting me know this matters.

**Comment by skarkkai** [comment]
Took a while but done now :)
